manifests/params.pp

```diff
@@ -36,14 +36,19 @@
     }
 
     'Debian': {
-      if versioncmp($::rubyversion, '2.1.0') >= 0 {
+      if ( ($::operatingsystem == 'Ubuntu') and (versioncmp($::operatingsystemrelease, '15.10') >= 0 ) ) {
         $ruby_pkg = 'ruby-augeas'
       }
-      elsif versioncmp($::rubyversion, '1.9.1') >= 0 {
-        $ruby_pkg = 'libaugeas-ruby1.9.1'
-      } else {
-        $ruby_pkg = 'libaugeas-ruby1.8'
-      }
+      else {
+        if versioncmp($::rubyversion, '2.3.0') == 0 {
+          $ruby_pkg = 'ruby-augeas'
```

Comment:
i'm confused… on both, Debian and Ubuntu the ruby version is >= 2.10 (2.3.0 ***is*** >= 2.1.0), and on *both* the package name is 'ruby'augeas'

so why do we need a special case for 2.3.0?